docs/custom-component.md

```diff
@@ -1,56 +0,0 @@
-Custom Components
----------
-
-In order to provide the flexibility necessary to build an advanced search experience, Vue Instant Search includes the ability to build custom components.
```

Comment:
I'm wondering if we should also have a sentence or 2 really make it clear that all provided components actually just "custom" components and these are all available as examples as well.